index.html

```diff
@@ -31,9 +31,21 @@
 
         let libView = new LibraryEntryPoint.LibraryView(configuration);
 
-        libView.on("itemClicked", function(item) {
+        libView.on("itemClicked", function (item) {
             console.log(item);
         })
+
+        libView.on("searchStarted", function (searchView) {
+            let searchText = searchView.getSearchText();
+            console.log(searchText);
+
+            // call search function (from web component) to return json.
+            // If json available, pass the json to search view.
+            // Otherwise pass null to search view, and search view will 
+            // run search locally
+            let searchResultItems = null;
+            searchView.updateSearchResultListItems(searchResultItems);
+        })
```

Comment:
If we just pass searchText to this event, and return the search result to libraryView instead of searchView, does it mean that we need to add one more attribute, like `searchResultItems`, to the configuration of libraryView?